apps/docs/content/components/date-range-picker/time-zones.ts

```diff
@@ -9,21 +9,23 @@ export default function App() {
           start: parseZonedDateTime("2024-04-01T00:45[America/Los_Angeles]"),
           end: parseZonedDateTime("2024-04-14T11:15[America/Los_Angeles]"),
         }}
+        label="Stay duration"
         labelPlacement="outside"
       />
       <DateRangePicker
         defaultValue={{
           start: parseAbsoluteToLocal("2024-04-01T07:45:00Z"),
           end: parseAbsoluteToLocal("2024-04-14T19:15:00Z"),
         }}
+        label="Stay duration"
         labelPlacement="outside"
       />
-  </div>
+    </div>
   );
 }`;
 
 const react = {
-  "/App.tsx": AppTs,
+  "/App.jsx": App,
```

Comment:
The code example was not displayed correctly.
https://nextui.org/docs/components/date-range-picker#time-zones

![image](https://github.com/nextui-org/nextui/assets/76232929/6bbb87d3-c33c-4308-89ab-fd3dc3097eda)

Reply:
Have you tried removing `node_modules` and install again? This is what I see in your PR branch. Only JS one. No TS.

<img width="1010" alt="image" src="https://github.com/nextui-org/nextui/assets/35857179/7809afb8-fe34-4207-ad05-9960dbb68575">

Reply:
@wingkwong 
That is expected. No TypeScript examples are shown, as can be seen in the other examples. The code before my fixes unnecessarily displayed broken TypeScript examples.